newm-server/src/main/resources/email/mint-released.html

```diff
@@ -0,0 +1,35 @@
+    <td align="left" style="font-size:0px;padding:0 0;padding-top:24px;word-break:break-word;">
+
+        <div style="font-family:Inter,sans-serif;font-size:14px;font-style:normal;font-weight:500;line-height:18px;text-align:left;color:#FFFFFF;">
+
+            Congrats [Fname]! Your track has been officially released and can now be found on all major streaming platforms.
+            <br><br>
+            [embed song, ISRC number & ownership share details]
```

Comment:
I'm curious about the format of these parameters @wlara and how to pass them to the form

Reply:
@saif-software-developer We put the same info in the other emails (e.g., started). All you have to do is add the placeholders `{song}` and `{collabs}`. Look at `min-started.mjml` for the formatting, ect.

Reply:
The code to replace the placeholders is already in-place. So, no other changes are needed.

Reply:
Got it , thanks :)